nuxt.config.js

```diff
@@ -109,15 +111,6 @@ export default {
     id: 'GTM-56W9ZCR',
   },
 
-  // Hooks configuration - https://content.nuxtjs.org/advanced/
-  hooks: {
-    'content:file:beforeInsert': (document) => {
-      if (document.extension === '.md') {
-        const stats = require('reading-time')(document.text)
-        document.readingStats = stats
-      }
-    },
-  },
```

Comment:
How are we handling reading time now?

Reply:
When fetching the data at that time we are calculating the reading time using the `reading-time` package.